pywwt/nbextension/static/time_test.ipynb

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
Is this file temporary?

Reply:
Yes. I made it so we could all work from the same test and see if my commits work for everyone or are specific to my set-up (especially with web browsers). I'll remove it once we have a working solution.